src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -60,6 +62,14 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return commandResult;
     }
 
+    private Model getAppropriateModel(Command command) {
```

Comment:
Thanks for abstracting this out for base Model.
We'll follow this moving onwards 👍